library/ssl_cli.c

```diff
@@ -1232,6 +1244,7 @@ static int ssl_parse_supported_point_formats_ext( mbedtls_ssl_context *ssl,
     }
 
     MBEDTLS_SSL_DEBUG_MSG( 1, ( "no point format in common" ) );
+        mbedtls_ssl_send_fatal_handshake_failure( ssl );
```

Comment:
Check the return value, to do what? Emitting a log is the job of lower-level functions. We don't want to return an error when sending an alert fails in these contexts: we're already returning the error that triggered the alert.